File: springbootjparest/src/main/java/at/eder/springbootjparest/models/Event.java
package at.eder.springbootjparest.models;

import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.experimental.SuperBuilder;

import javax.persistence.*;
import java.time.LocalDate;

@Data
@Entity
@NoArgsConstructor
@SuperBuilder
public class Event {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private String label;
    private LocalDate date;
    private double defaultPrice;

    @ManyToOne
    private Admin organiser;
}

File: WTMobileApp/app/src/main/java/at/eder/wirtschaftstagmobileapp/models/Event.kt
package at.eder.wirtschaftstagmobileapp.models

data class Event (
    var id: Long?,
    var label: String?,
    var date: String?,
    var organiser: User?
) {
    override fun toString(): String {
        return if (id == (-1).toLong())
            label!!
        else
            "$id - $label: $date"
    }
}
